### ISSUE TITLE
impossible to place blocks when schematics blocks are on

### ISSUE THREAD

**Issue by hitakiller** [maruohon/litematica#2]
I don't know how to explain this issue, 
when i've got a wrong block in my loaded schematic,it's impossible to place a block on this block, like the box of the red wrong block disallow me to place a block,
in the same way i can't place a block through a ghost schematic block down face.
For example, if i try to place a block on a grass_block through a schematic block, it does nothing.

Sorry for my english, i'm french so i don't know how to explain in goods terms this issue

**Comment by maruohon** [comment]
There are two possible causes for this:
* You have the Pick Block Last keybind set, and Pick Block enabled, so it will change the item in hand when you try to place a block through schematic blocks
* You have the Placement Restriction mode enabled, which tries to prevent wrongly placed blocks

**Comment by hitakiller** [comment]
yes i've both, but i only try to place the correct block at this point and
it seems like the schematic block is solid and prevent to place something
on like placing a torch on a glass block.

Le dim. 16 déc. 2018 à 18:07, Matti Ruohonen <notifications@github.com> a
écrit :
